microsoft-365/security/defender-endpoint/linux-install-manually.md

```diff
@@ -192,7 +192,7 @@ In order to preview new features and provide early feedback, it is recommended t
 - Install the Microsoft GPG public key:
 
     ```bash
-    curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+	curl -sSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/microsoft.gpg > /dev/null
     ```
 
 - Install the https driver if it's not already present:
```

Comment:
```suggestion
- Install the HTTPS driver if it's already present:
```

Reply:
Hi @JohanFreelancer9 I believe "Install the HTTPS driver if it's not already present:" to be gramatically correct. Since "not" is needed to imply:- install if you don't already have it on your system.